Datamigratie.Common/Services/Det/DetApiClient.cs

```diff
@@ -129,7 +145,15 @@ public async Task<DetZaak> GetSpecificZaakAsync(string zaaktypeId)
                 response.EnsureSuccessStatusCode();
 
                 var jsonString = await response.Content.ReadAsStringAsync();
-                return JsonSerializer.Deserialize<DetZaak>(jsonString, _options);
+                var result = JsonSerializer.Deserialize<DetZaak>(jsonString, _options);
+
+                if (result == null)
+                {
+                    logger.LogError("Failed to deserialize response from endpoint {Endpoint}", endpoint);
```

Comment:
het wordt een beetje muggenziften, maar feitelijk is het deserialiseren niet gefaald. Als dat het geval was, dan was er op regel 148 al een exception ontstaan en dan kwam je hier niet. 
Het is ook eigenlijk raar dat result null kan zijn, maar dat kan gebeuren als jsonString de string "null" bevat. 
Is het niet beter om hier gewoon te accepteren dat de uitkomst null kan zijn en dat in de aanroepende functies af te handelen, zoals je ook al netjes doet bij GetPagedData?